graphql_api/schema.py

```diff
@@ -46,6 +47,7 @@
 from graphql_api.mutation.tokenization import TokenizationMutation
 from graphql_api.mutation.weak_supervisor import WeakSupervisionMutation
 from graphql_api.mutation.zero_shot import ZeroShotMutation
+from graphql_api.mutation.personal_access_token import PersonalAccessTokenMutations
```

Comment:
the other mutations are called Mutation in singular and not Mutations. I suggest to stick to that naming

Reply:
renamed